src/PhpSpec/ObjectBehavior.php

```diff
@@ -31,9 +31,90 @@
  * @method void beConstructedWith(...$arguments)
  * @method void beConstructedThrough($factoryMethod, array $constructorArguments = array())
  * @method void beAnInstanceOf($class)
+ *
  * @method void shouldHaveType($type)
+ * @method void shouldNotHaveType($type)
+ * @method void shouldReturnAnInstanceOf($type)
```

Comment:
If I wanted to hard-code a list of matchers into the Subject, we'd add methods! We're never going to be able to annotate matchers that extensions add, or that users add in getMatchers...